pkg/framework/framework.go

```diff
@@ -124,14 +124,22 @@ func LoadClientset() (*kubernetes.Clientset, error) {
 	return kubernetes.NewForConfig(cfg)
 }
 
-func IsStatusAvailable(client runtimeclient.Client, name string) bool {
+func WaitForStatusAvailableShort(client runtimeclient.Client, name string) bool {
+	return expectStatusAvailableIn(client, name, WaitShort)
+}
+
+func WaitForStatusAvailableLong(client runtimeclient.Client, name string) bool {
```

Comment:
Since this uses wait medium let's call it medium
```suggestion
func WaitForStatusAvailableMedium(client runtimeclient.Client, name string) bool {
```